docs-ref-services/postgresql-mysql.md

```diff
@@ -54,46 +54,50 @@ Please see the [Python SDK authentication](https://docs.microsoft.com/python/azu
 
 #### Example
 
-Create a MySQL 5.7 Database resource and restrict access to a range of IP addresses using a firewall rule.
+Creates an Azure Database for MySQL server and restricts access to a range of IP addresses using a firewall rule.
```

Comment:
Should server be Server?

Reply:
We use lower case since "Server" is not part of the service name.